src/components/Toggle/Toggle.scss

```diff
@@ -0,0 +1,99 @@
+$toggle-track-length: rem(46px);
+$toggle-track-height: rem(24px);
+$toggle-thumb-size: rem(20px);
+$toggle-disabled-opacity: 0.6;
+$toggle-thumb-displacement: rem(2px);
+$toggle-border-size: rem(1px);
+$toggle-border-radius: rem(30px);
+$toggle-icon-adjusted-displacement: rem(2px);
+$toggle-icon-adjusted-size: rem(14px);
+
+.Toggle {
+  display: flex;
+  align-items: center;
+  border: $toggle-border-size solid transparent;
+
+  &.disabled {
+    opacity: $toggle-disabled-opacity;
+  }
+}
+
+.ToggleTrack {
+  width: $toggle-track-length;
+  height: $toggle-track-height;
+  border-radius: $toggle-border-radius;
+  box-shadow: 1px 1px 1px color('ink', 'lightest') inset;
+  background-color: color('sky');
+  transition: background-color duration(slow), border duration(slow),
+    box-shadow duration(slow);
+  -webkit-tap-highlight-color: transparent;
```

Comment:
@danrosenthal https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-tap-highlight-color

I tested it on my phone and it doesn't make a difference, so I will remove it.

Reply:
Incidentally, it looks pretty sexy on my phone. 😍